Sophie_wdl/pilon_practice.wdl

```diff
@@ -0,0 +1,33 @@
+version 1.0
+
+workflow pilon_practice {
+  input {
+    File reference
+    File bam_file
+    File bai_file
+    String message
+  }
+
+  call pilon_work { input: reference=reference, bam_file=bam_file, bai_file=bai_file, message=message}
+}
+
+task pilon_work {
+  input {
+    File reference
+    File bam_file
+    File bai_file
+    String message
+  }
+  command {
+    echo '${message}!'
+    pilon --genome ~{reference} --frags ~{bam_file} --variant --outdir . #this puts the files in cromwell-executions/pilon_practice/(most recent run folder)/call-pilon_work/execution/
```

Comment:
As said on Slack: you'd probably need to include `--tracks` here to make sure all additional outputs are generated!

Other tip: if you specify a certain amount of memory in the runtime block below, add `-Xmx{mem}G` to make sure Java allocates enough memory.